sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureActiveDirectoryConfiguration.java

```diff
@@ -172,4 +206,21 @@ protected void configure(HttpSecurity http) throws Exception {
                 .oidcUserService(oidcUserService);
         }
     }
+
+    /**
+     * Sample configuration to make RefreshTokenGrantRequestEntityConverter take effect.
```

Comment:
It's not `sample`, this code will take effect. I prefer the following description:
```
Temp solution to make RefreshTokenGrantRequestEntityConverter take effect.
```